index.js

```diff
@@ -4,15 +4,45 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "purple";
 
 
 /***** Deliverable 3 *****/
+const div = document.createElement('div');
+div.id = "profile"
+
+div.innerHTML = `<div id="profile">
+                <img src="${traveler.photo}"
+                alt="${traveler.name}">
+                <h2>${traveler.name}</h2>
+                <em>${traveler.nickname}</em>
+                </div>`;
+
+const travelerDiv = document.querySelector("div.traveler");
+travelerDiv.prepend(div);
+
 // console.log('This is what the traveler object looks like: ', traveler)
 
 
 /***** Deliverable 4 *****/
+const animal = document.createElement('li');
+
+traveler.animalSightings.forEach( function(sighting) {
+    animal.innerHTML += `<li data-id="${sighting.id}">
+    <p>${sighting.description}</p>
+    <img src="${sighting.photo}" alt="${sighting.species}"/>
+    <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>
+    </li>`
+});
+
+const animalsUl = document.querySelector('ul#animals');
+
+animalsUl.append(animal);
 
```

Comment:
You're on the right track! Currently you're creating an `li` and then adding a bunch of other `li`s within the `li`, so the parent of all the animal `li`s is another `li`.  You need to create [an `li`](https://www.w3schools.com/tags/tag_li.asp) for _each_ animal sighting post and the parent should be the `animalsUl`. You have several options: 

- Change the `innerHTML` of the `animalsUl`
```js
const animalsUl = document.querySelector('ul#animals');
animalsUl.append(fragment);

traveler.animalSightings.forEach( function(sighting) {
    animalsUl.innerHTML += `<li data-id="${sighting.id}">
    <p>${sighting.description}</p>
    <img src="${sighting.photo}" alt="${sighting.species}"/>
    <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>
    </li>`
});
```

- Using [document fragment](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment) (more advanced)
```js
const fragment = document.createDocumentFragment();;

traveler.animalSightings.forEach( function(sighting) {
    fragment.innerHTML += `<li data-id="${sighting.id}">
    <p>${sighting.description}</p>
    <img src="${sighting.photo}" alt="${sighting.species}"/>
    <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>
    </li>`
});

const animalsUl = document.querySelector('ul#animals');
animalsUl.append(fragment);
```